app/cdap/components/AddTransformation/index.tsx

```diff
@@ -14,19 +14,28 @@
  * the License.
  */
 
-import { Button, Container } from '@material-ui/core';
+import { Container } from '@material-ui/core';
 import DrawerWidget from 'components/DrawerWidget';
+import DirectiveContent from 'components/GridTable/components/DirectiveComponents';
+import { CALCULATE_OPTIONS } from 'components/GridTable/components/NestedMenu/menuOptions/calculateOptions';
```

Comment:
we should not have the calculate options in parsing, once we pull the updated add transformation step panel branch, we might have to recheck this - "we should have data and options related to current branch feature and change data type in our code, all others will only exist in their respective branches"